content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -93,6 +95,66 @@ For advanced usage of reordering rules, learn about the following reordering rul
 - :ref:`Visibility days <inventory/product_management/visibility-days>`
 - :ref:`Route <inventory/product_management/route>`
 
+0/0/1 reordering rule
+---------------------
+
+The *0/0/1* reordering rule is a specialty rule used to replenish a product that is not kept
+on-hand, each time a sales order (SO) is confirmed for that product.
+
+.. important::
+   The 0/0/1 reordering rule is similar to the *Replenish on Order (MTO)* route, in that both
+   workflows are used to replenish a product upon confirmation of an |SO|.
+
+   The main difference between the two methods is that the Replenish on Order route automatically
```

Comment:
Italicize the route's name since you are referencing it exactly :)
```suggestion
   The main difference between the two methods is that the *Replenish on Order* route automatically
```